common/text-level-modifiers/text-level-modifiers-en.html

```diff
@@ -0,0 +1,147 @@
+		</div>
+		<div class="row">
+			<p class="text-right col-sm-6">
+				Aligned right
+			</p>
+			<pre class="text-right col-sm-6"><code>&lt;p <strong>class=&quot;text-right&quot;</strong>&gt;...&lt;/p&gt;</code></pre>
+		</div>
+		<div class="row">
+			<p class="text-center col-sm-6">
+				Aligned center
+			</p>
+			<pre class="text-center col-sm-6"><code>&lt;p <strong>class=&quot;text-center&quot;</strong>&gt;...&lt;/p&gt;</code></pre>
+		</div>
+		<div class="row">
+			<p class="text-justify col-sm-6">
+				Justified text
+			</p>
+			<pre class="text-justify col-sm-6"><code>&lt;p <strong>class=&quot;text-justify&quot;</strong>&gt;...&lt;/p&gt;</code></pre>
+		</div>
+		<div class="row">
+			<p class="text-nowrap col-sm-6">
+				No-wrap text: xxxxxxxxxxxxxxxxxxxxxxx
+			</p>
+			<pre class="text-nowrap col-sm-6"><code>&lt;p <strong>class=&quot;text-danger&quot;</strong>&gt;...&lt;/p&gt;</code></pre>
+		</div>
+
+		<h2>Transformation examples:</h2>
+		<div class="row">
+			<p class="text-lowercase col-sm-6">
+				LOWERCASE TEXT
+			</p>
+			<pre class="col-sm-6"><code>&lt;p <strong>class=&quot;text-lowercase&quot;</strong>&gt;LOWERCASE TEXT&lt;/p&gt;</code></pre>
+		</div>
+		<div class="row">
+			<p class="text-uppercase col-sm-6">
+				uppercase text
+			</p>
+			<pre class="col-sm-6"><code>&lt;p <strong>class=&quot;text-uppercase&quot;</strong>&gt;uppercase text&lt;/p&gt;</code></pre>
+		</div>
+		<div class="row">
+			<p class="text-capitalize col-sm-6">
+				capitalized text
+			</p>
+			<pre class="col-sm-6"><code>&lt;p <strong>class=&quot;text-capitalize&quot;</strong>&gt;capitalized text&lt;/p&gt;</code></pre>
+		</div>
+	</div>
+	<div>
+		<h2><code>.no-wrap</code></h2>
+		<dd>Prevents the text from wrapping</dd>
+		<p class="no-wrap">
+			Example no wrap: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+		</p>
+		<pre><code>&lt;p <strong>class=&quot;no-wrap&quot;</strong>&gt;...&lt;/p&gt;</code></pre>
+	</div>
+	<div>
+		<h2><code>.fnt-nrml</code></h2>
+		<dd>Resets the font size to the default</dd>
+		<p>Example 1: Default font style</p>
+		<p class="fnt-nrml">Example 2: Normal font style</p>
+		<p><strong>Example 3: <em class="fnt-nrml">Normal font style</em></strong></p>
+		<p><em>Example 4: <strong class="fnt-nrml">Normal font style</strong></em></p>
+		<pre><code>&lt;p&gt;Example 1: Default font style&lt;/p&gt;</code></pre>
+		<pre><code>&lt;p <strong>class=&quot;fnt-nrml&quot;</strong>&gt;Example 2: Normal font style&lt;/p&gt;</code></pre>
+		<pre><code>&lt;p <strong>class=&quot;fnt-nrml&quot;</strong>&gt;&lt;strong&gt;Example 3: &lt;em class="fnt-nrml"&gt;Normal font style&lt;/em&gt;&lt;/strong&gt;&lt;/p&gt;</code></pre>
+		<pre><code>&lt;p <strong>class=&quot;fnt-nrml&quot;</strong>&gt;&lt;em&gtExample 4: &lt;strong class="fnt-nrml"&gtNormal font style&lt;/strong&gt&lt;/em&gt&lt;/p&gt;</code></pre>
```

Comment:
Put it in one block like

```suggestion
		<pre><code>&lt;p&gt;Example 1: Default font style&lt;/p&gt;
&lt;p <strong>class=&quot;fnt-nrml&quot;</strong>&gt;Example 2: Normal font style&lt;/p&gt;
&lt;p <strong>class=&quot;fnt-nrml&quot;</strong>&gt;&lt;strong&gt;Example 3: &lt;em class="fnt-nrml"&gt;Normal font style&lt;/em&gt;&lt;/strong&gt;&lt;/p&gt;
&lt;p <strong>class=&quot;fnt-nrml&quot;</strong>&gt;&lt;em&gtExample 4: &lt;strong class="fnt-nrml"&gtNormal font style&lt;/strong&gt&lt;/em&gt&lt;/p&gt;</code></pre>
```